bot/handlers/callbacks.py

```diff
@@ -77,7 +92,13 @@ async def select_peer_callback(callback: CallbackQuery, callback_data: Connectio
 )
 async def ban_user_callback(callback: CallbackQuery, callback_data: UserActionsCallbackData):
     client = ClientFactory(tg_id=callback_data.user_id).get_client()
+    peers = client.get_peers()
     client.set_status(ClientStatusChoices.STATUS_ACCOUNT_BLOCKED)
+    # if peer has no peers, it will raise KeyError, so we suppress it
+    with suppress(KeyError):
```

Comment:
it will not raise KeyError. `for` cycles over empty lists are no-ops. So suppressing is unnecessary here